jekyll/_cci2/examples.md

```diff
@@ -4,6 +4,13 @@ title: "Testing Config Files Locally"
 description: "Example CircleCI 2.0 Configuration Files"
 ---
 
+
+<div class="alert alert-info" role="alert">
+<b>Note:</b> This document describes a legacy means of testing your config file locally.
```

Comment:
I would say 'alternate' means instead of legacy here, and also link from the CLI document to this one for those who need an alternate method like this one.